docs/5.0/enterprise/ssh-rbac.md

```diff
@@ -133,6 +133,11 @@ spec:
       # of the list example above can be expressed as:
       'environment': '^test|staging$'
 
+    kubernetes_labels:
+    - env: prod
+    - region: us-west1
+    - cluster_name: '^us.*\.example\.com$' # cluster_name label is generated automatically
```

Comment:
I'm not sure this comment makes sense in the context; if `cluster_name` is generated automatically, why are we listing it?